modules/sdk-coin-sol/src/sol.ts

```diff
@@ -110,11 +110,11 @@ interface SolDurableNonceFromNode {
 }
 
 interface RecoveryOptions {
-  userKey: string; // Box A
-  backupKey: string; // Box B
+  userKey?: string; // Box A
+  backupKey?: string; // Box B
   bitgoKey: string; // Box C - this is bitgo's xpub and will be used to derive their root address
   recoveryDestination: string; // base58 address
-  walletPassphrase: string;
+  walletPassphrase?: string;
   durableNoncePK?: string;
   durableNonceSK?: string;
```

Comment:
just caught this, but since these two parameters need to be present together, can we group them in a single object?
i.e.
```
interface RecoveryOptions {
    ...
    durableNonce?: {
        publicKey: string;
        secretKey: string;
    }
}
```

this way, you just need to check for the existence of `durableNonce`. if its present, then so should `durableNonce.publicKey` and `durableNonce.secretKey`